packages/hmr-plugin/src/internal/hmr-options-builder.ts

```diff
@@ -1,4 +1,7 @@
+import { isJIT } from '@ngxs/store/internals';
+
 import { NgxsHmrOptions } from '../symbols';
+import { ivyEnabledInDevMode } from '../../../store/src/ivy/ivy-enabled-in-dev-mode';
```

Comment:
This relative reference is causing the build to fail.
You need to reference through the `@ngxs/...` packages.